strapi/strapi
# feat(content-releases): Release delete action

### What does it do?

Add the delete action in the Details page to delete a Release

### Why is it needed?

We need a way to delete a release from the Details page menu

### How to test it?

You can go to the Releases page and create new Release
- then in the Details page click on the "three dot" button and open the menu with the Delete button
- click the Delete button
- confirm the deletion of the Release in the Confirmation dialog
- you will be redirected to the Releases list page
